src/app/tic-tac-toe.component.scss

```diff
@@ -55,9 +55,9 @@ button:disabled {
 }
 
 .gameCell {
-  background: #FAFAFA;
+  background: #FAFAFA; // review
```

Comment:
Consider using SCSS variables for (potentially ~) repetitive values such as colors. It makes future changes easier -- instead of replacing all occurrences, you just redefine the variable.

http://sass-lang.com/guide